pkg/operator/controllers/ingress/ingress_controller_test.go

```diff
@@ -105,6 +123,8 @@ func TestReconciler(t *testing.T) {
 				},
 			},
 			expectedReplica: minimumReplicas,
+			startConditions: defaultConditions,
+			wantConditions:  defaultConditions,
```

Comment:
it looks like every test case is this - let's add at least one test for the code changes you've made - that errors do cause conditions to change (and ideally, one test case for every possible error).

Reply:
Changes made. Thanks!